### DISCUSSION TITLE
Comments

### DISCUSSION THREAD

**Discussion by boberski666** [mediacms-io/mediacms#843] [Q&A] [answered]
Is there a way to add comments for visitores without login? Like some approve type

**Comment by mgogoulos** [comment, marked as answer]
You can't with the current version, you would need to develop that functionality, because now a Comment is tight to a user. So this relationship needs to be loosened, and also provide some more fields on the comments (eg name or email, since it is supposed to get them through the User  that is logged in, so if you allow for anonymous users you would want to have this info)...
